stable/msoms/templates/NOTES.txt

```diff
@@ -0,0 +1,34 @@
+** Please be patient while the chart is being deployed **
```

Comment:
I don't think this is needed. At this point the chart should already be deployed